### PR TITLE
[TP-4] create GET request on all tables

### PR DESCRIPTION
**Pull request by GOSTT22**
I am complated GET request on all tables
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: routes/education.js [hunk 1]
```diff
@@ -28,4 +28,22 @@ router.post("/", (req, res) => {
...
+
+  let sql = `select * from education`;
+
+  db.query(sql, (error, result) => {
+    if (error) {
+      console.error(error);
+      throw error;
+    }
+    if (!result.length) {
+      console.log("Table : education is empty");
```
Comment (buba3230) [inline routes/education.js:42]: Who will be see that console log?  If the table is empty - user should get some data in response...

File: routes/placement.js [hunk 2]
```diff
@@ -83,4 +83,78 @@ router.post("/nameStreet", (req, res) => {
...
+
+  let sql = `select * from state`;
+
+  db.query(sql, (error, result) => {
+    if (error) {
+      console.error(error);
+      throw error;
+    }
+    if (!result.length) {
+      console.log("Table : state is empty");
```
Comment (buba3230) [inline routes/placement.js:98]: And here

File: routes/placement.js [hunk 3]
```diff
@@ -83,4 +83,78 @@ router.post("/nameStreet", (req, res) => {
...
+
+  let sql = `select * from street`;
+
+  db.query(sql, (error, result) => {
+    if (error) {
+      console.error(error);
+      throw error;
+    }
+    if (!result.length) {
+      console.log("Table : street is empty");
```
Comment (buba3230) [inline routes/placement.js:152]: The same

**Review by buba3230 — CHANGES_REQUESTED** [review]

**Review by StayTeeVone — APPROVED** [review]
all rigth!

**Review by buba3230 — APPROVED** [review]

**Review by StayTeeVone — APPROVED** [review]

**Review by OleynikBogdan — APPROVED** [review]

**Review by Shapovall — APPROVED** [review]

**Pull request merged**